miqa/core/rest/frame.py

```diff
@@ -81,7 +80,12 @@ class Meta:
         fields = ['content', 'scan']
 
 
-class FrameViewSet(ListModelMixin, GenericViewSet, mixins.CreateModelMixin):
+class FrameViewSet(
+    mixins.ListModelMixin,
+    mixins.RetrieveModelMixin,
+    GenericViewSet,
+    mixins.CreateModelMixin,
```

Comment:
I think this goes before `GenericViewSet`, although I confess I'm not entirely sure what it would break.